pkg/liqoctl/install/interface.go

```diff
@@ -0,0 +1,20 @@
+package install
+
+import (
+	"context"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/client-go/rest"
+)
+
+// ProviderInterface defines the methods required to support the Liqo install for a given provider.
+type ProviderInterface interface {
+	// ValidateCommandArguments validates the flags passed as arguments to the install command
+	ValidateCommandArguments(*flag.FlagSet) error
+	// ExtractChartParameters retrieves the install parameters required for a correct installation. This may require to
+	// instantiate extra clients to interact with cloud provider or the target cluster.
+	ExtractChartParameters(context.Context, *rest.Config) (string, error)
```

Comment:
```suggestion
	ExtractChartParameters(context.Context, *rest.Config) error
```

Since it is not used